.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+orbs:
+  node: circleci/node@5.0.2
```

Comment:
input the latest node version

Reply:
5.0.2 is the latest according to the orbs registry